Dockerfile

```diff
@@ -13,4 +16,4 @@ COPY requirements.txt /
 RUN pip install --no-cache-dir -r /requirements.txt
 
 WORKDIR /app
-COPY management_interface /app
\ No newline at end of file
+COPY management_interface /app
```

Comment:
I *think* this should actually be:
```suggestion
COPY management_interface/management_interface /app
```
Is there a reason we need the outer directory in the container?

Reply:
I'll patch this.  It obsoletes #2 but that's fine.